internal/controller/constants.go

```diff
@@ -36,34 +36,54 @@ const (
 	AppServerMetricsPath = "/metrics"
-
-	// Image of the OLS application redis server
-	//RedisServerImageDefault = "quay.io/openshift/lightspeed-service-redis:latest"
 	// OLSConfigHashKey is the key of the hash value of the OLSConfig configmap
 	OLSConfigHashKey = "hash/olsconfig"
 	// LLMProviderHashKey is the key of the hash value of OLS LLM provider credentials consolidated
 	// #nosec G101
 	LLMProviderHashKey = "hash/llmprovider"
-	// RedisConfigHashKey is the key of the hash value of the OLS's redis config
-	RedisConfigHashKey = "hash/olsredisconfig"
-	// RedisSecretHashKey is the key of the hash value of OLS Redis secret
+	// PostgresDefaultUser is the default user name for postgres
+	PostgresDefaultUser = "postgres"
+	// PostgresDefaultDbName is the default db name for postgres
+	PostgresDefaultDbName = "postgres"
+	// PostgresConfigHashKey is the key of the hash value of the OLS's postgres config
+	PostgresConfigHashKey = "hash/olspostgresconfig"
+	// PostgresSecretHashKey is the key of the hash value of OLS Postgres secret
+	// #nosec G101
+	PostgresSecretHashKey = "hash/postgres-secret"
+	// PostgresServiceName is the name of OLS application postgres server service
+	PostgresServiceName = "lightspeed-postgres-server"
+	// PostgresSecretName is the name of OLS application postgres secret
+	PostgresSecretName = "lightspeed-postgres-secret"
+	// PostgresCertsSecretName is the name of the postgres certs secret
+	PostgresCertsSecretName = "lightspeed-postgres-certs"
+	// PostgresBootstrapSecretName is the name of the postgres bootstrap secret
 	// #nosec G101
-	RedisSecretHashKey = "hash/redis-secret"
-	// RedisServiceName is the name of OLS application redis server service
-	RedisServiceName = "lightspeed-redis-server"
-	// RedisSecretName is the name of OLS application redis secret
-	RedisSecretName = "lightspeed-redis-secret"
-	// OLSAppRedisCertsName is the name of the OLS application redis certs secret
-	RedisCertsSecretName = "lightspeed-redis-certs"
+	PostgresBootstrapSecretName = "postgres-bootstrap"
+	// PostgresBootstrapVolumeMount is the name of bootstrap volume mount
+	PostgresBootstrapVolumeMount = "/usr/share/container-scripts/postgresql/start/create-extensions.sh"
+	// PostgresExtensionScript is the name of the postgres extensions script
+	PostgresExtensionScript = "create-extensions.sh"
+	// PostgresConfigMap is the name of the postgres config map
+	PostgresConfigMap = "postgres-conf"
+	// PostgresConfigVolumeMount is the name of postgres configuration volume mount
+	PostgresConfigVolumeMount = "/usr/share/pgsql/postgresql.conf.sample"
+	// PostgresConfig is the name of postgres configuration used to start the server
+	PostgresConfig = "postgresql.conf.sample"
+	// PostgresDataVolume is the name of postgres data volume
+	PostgresDataVolume = "postgres-data"
+	// PostgresDataVolumeMount is the name of postgres data volume mount
+	PostgresDataVolumeMount = "/var/lib/pgsql/data"
```

Comment:
doesn't look like a volume mount name, do you mean path?

Reply:
Renamed as path